packages/dev-server-esbuild/package.json

```
@@ -52,7 +52,7 @@
   "dependencies": {
     "@mdn/browser-compat-data": "^4.0.0",
     "@web/dev-server-core": "^0.3.17",
-    "esbuild": "^0.12.21",
+    "esbuild": "^0.12|^0.13|^0.14",
```

I think this should be like this:

```suggestion
    "esbuild": "^0.12 || ^0.13 || ^0.14",
```

---

But I this works better:

```suggestion
    "esbuild": ">=0.12",
```

(but maybe we should update it in a future breaking changes)

---

Yes ofc it should be `||`… I have changed that.

I can also change it to `>=0.12` if everybody is ok with that.